moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua

```diff
@@ -0,0 +1,147 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Mozilla Security Cloudtrail notifications
+
```

Comment:
Some more details on what the sandbox does would be good here to be consistent with the others

Reply:
fixed